src/scan.ts

```diff
@@ -47,9 +49,10 @@ export async function scanComponents (dirs: ScanDir[]): Promise<Component[]> {
       const pascalName = pascalCase(fileName)
       const kebabName = kebabCase(fileName)
 
+      const meta = { filePath, pascalName, kebabName }
       components.push(
-        prefixComponent(prefix, { pascalName, kebabName, import: `require('${filePath}').default` }),
-        prefixComponent(LAZY_PREFIX, prefixComponent(prefix, { pascalName, kebabName, import: `function () { return import('${filePath}' /* webpackChunkName: "components/${kebabName}" */) }` }))
+        prefixComponent(prefix, { ...meta, import: `require('${filePath}').default` }),
+        prefixComponent(LAZY_PREFIX, prefixComponent(prefix, { ...meta, async: true, import: `function () { return import('${filePath}' /* webpackChunkName: "components/${kebabName}" */) }` }))
```

Comment:
Why not using `/* webpackChunkName: "${filePath}" */` ?

Reply:
Also removing `srcDir` and I guess the extension?

Reply:
Seems good idea. I was already doing this for generated meta json